Makefile

```diff
@@ -74,6 +74,9 @@ run: install
 html: install
 	. $(VENV); $(SPHINXBUILD) -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" -w $(SPHINXDIR)/warnings.txt $(SPHINXOPTS)
 
+check-html-build: install
+	. $(VENV); $(SPHINXBUILD) -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" -E --fail-on-warning --keep-going $(SPHINXOPTS)
+
```

Comment:
The `Makefile` file is taken from the template (sphinx-starter repo) and I think it's best to leave it untouched, so that we can always upgrade the template with less conflicts.

We can create a separate file (e.g., `Makefile.custom.mk`) and call it with `make -f Makefile.custom.mk check-html-build` from the CI job.

Reply:
I would agree, but I think we should have a good understanding of what's going on in the makefile so that we can make upgrades and changes without worrying about straying from the template. Although the best thing overall would be an improvement to the template, which I can propose.

Reply:
Yeah, it'd be great if you could propose it to the upstream repo. To be fair, the conflict wouldn't be a big issue.